src/Adobe-Glyph.xml

```diff
@@ -0,0 +1,37 @@
+<SPDX name="Adobe Glyph List License" identifier="Adobe-Glyph" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/MIT#AdobeGlyph</url>
+  </urls>
+  <license>
+    <copyright>
+      <p>Copyright (c) 1997,1998,2002,2007 Adobe Systems Incorporated</p>
+    </copyright>
+    <body>
+      <p>Permission is hereby granted, free of charge, to any person obtaining a copy of this documentation file
+         to use, copy, publish, distribute, sublicense, and/or sell copies of the documentation, and to permit
+         others to do the same, provided that:</p>
+      <list>
+        <li>
+          <b>-</b>
+          <p>No modification, editing or other alteration of this document is allowed; and</p>
+        </li>
+        <li>
+          <b>-</b>
+          <p>The above copyright notice and this permission notice shall be included in all copies of the
+             documentation.</p>
+        </li>
+      </list>
+      <p>Permission is hereby granted, free of charge, to any person obtaining a copy of this documentation file,
+         to create their own derivative works from the content of this document to use, copy, publish,
+         distribute, sublicense, and/or sell the derivative works, and to permit others to do the same,
+         provided that the derived work is not represented as being a copy or version of this document.</p>
+      <p>Adobe shall not be liable to any party for any loss of revenue or profit or for indirect, incidental,
+         special, consequential, or other similar damages, whether based on tort (including without limitation
+         negligence or strict liability), contract or other legal or equitable grounds even if Adobe has been
+         advised or had reason to know of the possibility of such damages.Ê The Adobe materials are provided on
```

Comment:
This is almost certainly a UTF-8 encoding artifact, but I'm not sure what character it was intended to be. Normally another character would follow it.

If we remove it, this license will probably not match other people who have copied the same text, but if we leave it it won't match people who fixed it or got the "correct" version from somewhere. It might deserve consideration for a new matching guideline dealing with ignoring high ascii characters, but that has potential problems with languages that need them. Definitely worth discussion?